src/Core/tests/DeviceTests/AssertionExtensions.Android.cs

```diff
@@ -45,7 +46,7 @@ public static AColor ColorAtPoint(this Bitmap bitmap, int x, int y, bool include
 			}
 		}
 
-		public static Bitmap ToBitmap(this AView view)
+		public static async Task<Bitmap> ToBitmap(this AView view)
```

Comment:
This needs to be async now as we need to add a little delay so that the view actually gets attached to the UI before we try draw it.